cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -1163,16 +1195,18 @@ d8bb2c8334e3b92a42c0078983e753c054e647ad5387ed149f802f52b5a95ebf
       / alg / 1: 10 / AES-CCM-16-64-128 /
     } >>,
     / unprotected / {
-      / iv / 5: h'86bbd41cc32604396324b7f380'
+      / iv /  5: h'86bbd41cc32604396324b7f380',
+      / kid / 4: 'Symmetric128'
     },
-    / ciphertext / h'72439fbff538aa7b601ebfb29454050a3c99fd13b27
-                     216d084556496c7355c4bb462510f8e0e8479dbe087
-                     22d620e96bcb7764d75140d96220f062679b46b897e
-                     7abe0c325dc2c96d8bb2c8334e3b92a42c0078983e7
-                     53c054e647ad5387ed149f802f52b5a95ebf5f153c4
-                     fd64854ab7531e082b7f22721f939d257c94f8bc248
-                     e1d9cf04f9dd4e5de7ab62df37842fabec230a657d4
-                     abf7162bc786345ebb8eb3af0'
+    / ciphertext / h'f6b0914f993de822cc47e5e57a188d7960b528a7474
+                     46fe12f0e7de05650dec74724366763f167a29c002d
+                     fd15b34d8993391cf49bc91127f545dba8703d66f5b
+                     7f1ae91237503d371e6333df9708d78c4fb8a8386c8
+                     ff09dc49af768b23179deab78d96490a66d5724fb33
+                     900c60799d9872fac6da3bdb89043d67c2a05414ce3
+                     31b5b8f1ed8ff7138f45905db2c4d5bc8045ab372bf
+                     f142631610a7e0f677b7e9b0bc73adefdcee16d9d5d
+                     284c616abeab5d8c291ce0'
```

Comment:
Why did the length of this change?

Reply:
because the signed object that is encrypted got a kid attribute too